web/urls.py

```diff
@@ -17,4 +17,5 @@
 	url(r'^privacy/$', views.privacy, name='pages-privacy'),
 	url(r'^project/(?P<project_id>[0-9]+)/$', views.project_detail),
 	url(r'^logout/$', views.logout, name='logout'),
+	url(r'^member_page/$', views.member_page, name='pages-member_page'),
```

Comment:
@ercchy funny, i had dashes first, but then i thought this is more correct, since Livia also used project_detail :) 

@livike i am trying to solve this dynamic view, according to yours but it does not work, any ideas? thx :)
url(r'^member_page/(?P<member_page>[a-z]+)/$', views.member_page),